languages/csharp/exercises/concept/chars/.docs/instructions.md

```diff
@@ -0,0 +1,41 @@
+In this exercise you will implement a partial set of utility routines to help a developer
+clean up identifier names.
+
+In the 4 tasks you will gradually build up the routine `Clean` A valid identifier comprises
+zero or more letters and underscores.
+
+In all cases the input string is guaranteed to be non-null. Note that the `Clean` method should treat an empty string as valid.
+
+### 1. Replace any spaces encountered with underscores
+
+This also applies to leading and trailing spaces.
```

Comment:
The tasks should specify what method to implement, and usually also include the task header's description in some format. Maybe something like this:

```suggestion
Implement the (_static_) `Identifier.Clean()` method to replace any spaces with underscores. This also applies to leading and trailing spaces.
```